Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:latest 
+
+ENV PG_USERNAME="postgres"
+ENV PG_PASSWORD=""
+ENV PG_DATABASE="2FactorAuth"
+ENV PG_HOSTNAME="127.0.0.1"
+ENV PG_PORT="32768"
+ENV GOPATH="/go"
```

Comment:
fwiw - im suprised youre setting this explictly - isn't it set from the base image?

Reply:
Yes it is. I was just trying to match my env to the image, but yeah, this statement is not really necessary. Ill take it out.